scripts/zones/Port_Jeuno/npcs/Gaura.lua

```diff
@@ -1,25 +1,26 @@
 -----------------------------------
 -- Area: Port Jeuno
 --  NPC: Guara
--- Standard Info NPC
 -----------------------------------
 local entity = {}
 
 entity.onTrade = function(player, npc, trade)
 end
 
 entity.onTrigger = function(player, npc)
-
     local vHour = VanadielHour()
     local vMin  = VanadielMinute()
 
     while vHour >= 3 do
         vHour = vHour - 6
     end
 
-    if (     vHour == -3) then vHour = 3
-    elseif ( vHour == -2) then vHour = 4
-    elseif ( vHour == -1) then vHour = 5
+    if vHour == -3 then
```

Comment:
Not to future self, this can be simplified to `vHour = vHour + 6` so long as we check for the appropriate conditions (would need to work through the math for why we're progressively subtracting 6 before)

Reply:
Yeah, I was maning to look at all this vHour scripts ina future PR andtry to simplify them, or even create a helper in zone script if I can find a common base